test/suites/build/build.go

```diff
@@ -1,22 +1,99 @@
 package build
 
 import (
+	"fmt"
 	"path/filepath"
+	"strconv"
+	"strings"
 
 	"github.com/alibaba/sealer/test/testhelper"
+	"github.com/alibaba/sealer/test/testhelper/settings"
+	"github.com/alibaba/sealer/utils"
+	"github.com/onsi/gomega"
 )
 
-func getFixtures() string {
-	pwd := testhelper.GetPwd()
-	return filepath.Join(pwd, "suites", "build", "fixtures")
+func GetFixtures() string {
+	return filepath.Join(testhelper.GetPwd(), "suites", "build", "fixtures")
 }
 
-func GetOnlyCopyDir() string {
-	fixtures := getFixtures()
-	return filepath.Join(fixtures, "only_copy")
+func GetLocalBuildDir() string {
+	return "local_build"
 }
 
-func GetBuildTestDir() string {
-	fixtures := getFixtures()
-	return filepath.Join(fixtures, "build_test")
+func GetCloudBuildDir() string {
+	return "cloud_build"
+}
+
+func GetTestKubeKile() string {
+	// only copy
+	return ""
+}
+
+func GetTestContext() string {
+	return ""
+}
+func GetTestBuildType() string {
+	return ""
+}
+
+func GetTestImageName() string {
+	return fmt.Sprintf("sealer-io/%s%d:%s", settings.ImageName, 719, "v1")
+}
+
+func GetTestClusterFilePath() string {
+	return ""
+}
+
+type ArgsOfBuild struct {
+	KubeFile, ImageName, Context, BuildType string
+}
+
+func (a *ArgsOfBuild) SetKubeFile(kubeFile string) *ArgsOfBuild {
```

Comment:
attribution could be set using "a.KubeFile = kubeFile", why encapsulate the operation?

Reply:
I think that's okay, because you don't know the set logic will change or not in the future.